File: faculdade/semestre-01/programacao-de-computadores/exercicios/aula-02-atribuicao-entrada-e-saida/exerc01.py
#Desenvolva um programa em python que solicite ao usario os valores dos lados de um retângulo e calcule e mostre seu perímetro e sua área

lado1 = int(input("Digite o valor do primeiro lado: "))
lado2 = int(input("Digite o valor do segundo lado: "))
perimetro = (lado1*2)+(lado2*2)
area = lado1*lado2

print ("O perimetro do rentângulo é: ", perimetro, ",sendo a sua área: ", area)

File: Java/exercicios-diversos/exercicios-diversos/src/exercicios/exerc01.java
//#Desenvolva um programa que solicite ao usario os valores dos lados de um retângulo e calcule e mostre seu perímetro e sua área
package exercicios;

import java.util.Scanner;

public class exerc01 {
    static void main() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Digite o primeiro lado: ");
        int lado1 = sc.nextInt();
        System.out.print("Digite o segundo lado: ");
        int lado2 = sc.nextInt();
        int calculoArea = lado1*lado2;
        int calculoPerimetro = 2*(lado1+lado2);

        System.out.println("A área do seu retângulo é: " + calculoArea + " e o perimetro é: "+ calculoPerimetro);
        sc.close();
    }

}
